components/prism-mongodb.js

```diff
@@ -0,0 +1,103 @@
+
+
```

Comment:
Please remove the empty lines.

Reply:
Already added this change as part of commits